### ISSUE TITLE
ngx-bootstrap-product-tour is getting error as ngx-bootstrap@1.9.3 is not supported with angular5

### ISSUE THREAD

**Issue by chinna-magapu** [NMilicic/ngx-bootstrap-product-tour#4]
Hi,  I am using ngx-bootstrap-product-tour with angular 5 it is working fine on the local machine.
when I run a Prod Build it throws the below error

```
ERROR in: "let-" is only supported on ng-template elements. ("
  </thead>
  <tbody>
    <template ngFor [ngForOf]="rows" [ERROR ->]let-rowz="$implicit" let-index="index">
      <tr *ngIf="!(datePicker.onlyCurrentMonth && rowz[0].sec")
: "let-" is only supported on ng-template elements. ("
  </thead>
  <tbody>
    <template ngFor [ngForOf]="rows" let-rowz="$implicit" [ERROR ->]let-index="index">
      <tr *ngIf="!(datePicker.onlyCurrentMonth && rowz[0].secondary && rowz[6].sec")

```

I researched about this error, it says  
ngx-bootstrap-product-tour is using ngx-bootstrap@1.9.3 but it is throwing an error in angular5

as per below link 

https://github.com/valor-software/ngx-bootstrap/issues/3024

saying that need to update ngx-bootstrap@1.9.3  to ngx-bootstrap@2.0.0-beta.8. 
is there any solution to update the dependency to make it work in angular5.

**Comment by NMilicic** [comment]
@chinna-magapu Should be fixed with #6 
I published it on npm under version 2.0.0 so please update your's package.json

**Comment by chinna-magapu** [comment]
Thanks, @NMilicic for your quick update. Issue got fixed for me
